src/core/components/Conversation.tsx

```diff
@@ -32,17 +33,11 @@ const ConversationComponent = ({ messages, onRendered }: ConversationProps) => {
   const { errorText, isRequesting, progressStore, messageStore, registry } =
     useAppContext();
 
-  const FEAT_UPDATED_DESIGN = import.meta.env.VITE_FEAT_UPDATED_DESIGN;
-  /* FEAT_UPDATED_DESIGN TODO: when this goes live:
-       - update the #conversation id in chat view to have:  var(--spacing-lg);
-       - remove style={ on the div}}
-    
-    */
   return (
     <div
       id={styles.conversation}
       data-testid="conversation"
-      style={{ marginTop: FEAT_UPDATED_DESIGN ? '24px' : '-10px' }}
+      style={{ marginTop: !isInIframe() ? '24px' : '-10px' }}
```

Comment:
my github comment above was not correct. We still need the negative margin in the iframe. Remove the css from the modules for marginTop and just apply it here as a style for now